.circleci/config.yml

```diff
@@ -335,7 +335,32 @@ jobs:
           registry: registry.heroku.com
           tag: latest
       - run: heroku container:release -a simple-hub-staging simple-hub
-
+      
+  deploy-persistent-seeder:
+    working_directory: /home/circleci/project
+    resource_class: large
+    docker:
+      - image: circleci/node:10.16.3-browsers
+    steps:
+      - setup_remote_docker
+      - checkout
+      - <<: *restore_dep
+      - <<: *restore_lib
+      - run: rm -rf ./patches && cd ./packages/ && find . -maxdepth 1 ! -path "./channel-provider" ! -path "./client-api-schema" ! -path "./e2e-tests" ! -path "./devtools" -exec rm -rf {} ';'
+      - run: yarn --frozen-lockfile
+      - run: curl https://cli-assets.heroku.com/install.sh | sh
+      - run: heroku container:login
+      - docker/build:
+          cache_from: registry.heroku.com/persistent-seeder-staging/persistent-seeder:latest
+          dockerfile: packages/e2e-tests/persistent-seeder-deployment/persistent-seeder.dockerfile
+          image: registry.heroku.com/persistent-seeder-staging/persistent-seeder
+          tag: latest
+      - docker/push:
+          image: persistent-seeder-staging/persistent-seeder
+          registry: registry.heroku.com
+          tag: latest
+      - run: heroku container:release -a persistent-seeder-staging persistent-seeder
```

Comment:
We don't _really_ have a staging environment. We just have one production environment. Maybe we shouldn't keep using "staging" in the names of things given this as it might cause confusion.

Reply:
My original thinking was staging == ropsten and production == mainnet.

Reply:
Should we consider labelling all other things as staging too then, e.g., the web3torrent and xstate-wallet deployments? I suppose the wallet is intended to support multiple chains... worth a discussion

Reply:
On second thought, should we get rid of staging/production terminology and instead use ropsten/mainnet terminology? So the heroku application name becomes `persistent-seeder-ropsten`

Reply:
Yeah I think that makes more sense. It's not clear that we need separate environments or deployments for both chains though. A single service might be able to work in production against multiple chains.

Reply:
I don't know the specifics, but I think that's out of scope for this PR.
To start with, to support multiple chains, we need to change code to do so, right now the chain is a single string, not an array of options. 
...At least in web3torrent, I don't think this suggestion would be a one-line change.